verilog/formatting/tree_unwrapper_test.cc

```diff
@@ -4428,6 +4429,202 @@ TEST_F(TreeUnwrapperTest, UnwrapSequenceTests) {
+        NL(1, {"reg", "o", ";"}),
+        NL(1, {"input", "c", ";"}),
+        NL(1, {"input", "d", ";"}),
+        UdpBody(1, L(1, {"table"}),
+                   NL(2, {"(01)", "0", ":", "?", ":", "0", ";"}),
+                   NL(2, {"(01)", "1", ":", "?", ":", "1", ";"}),
+                   NL(2, {"(0?)", "1", ":", "1", ":", "1", ";"}),
+                   NL(2, {"(0?)", "0", ":", "0", ":", "0", ";"}),
+                   NL(2, {"(?0)", "?", ":", "?", ":", "-", ";"}),
+                   NL(2, {"?", "(??)", ":", "?", ":", "-", ";"}),
+                   L(1, {"endtable"})
+               ),
+        L(0, {"endprimitive"}),
+    },
+    {
+        "mixed sensitivity sequential UDP",
+        "primitive mixed(o, clk, j, k, preset, clear); "
+        "  output o; "
+        "  reg o; "
+        "  input c; "
+        "  input j, k; "
+        "  input preset, clear; "
+        "  table "
+        "    ?  ??  01  : ? :  1 ; "
+        "    ?  ??  *1  : 1 :  1 ; "
+        "    ?  ??  10  : ? :  0 ; "
+        "    ?  ??  1*  : 0 :  0 ; "
+        "    r  00  00  : 0 :  1 ; "
+        "    r  00  11  : ? :  - ; "
+        "    r  01  11  : ? :  0 ; "
+        "    r  10  11  : ? :  1 ; "
+        "    r  11  11  : 0 :  1 ; "
+        "    r  11  11  : 1 :  0 ; "
+        "    f  ??  ??  : ? :  - ; "
+        "    b  *?  ??  : ? :  - ; "
+        "    b  ?*  ??  : ? :  - ; "
+        "  endtable "
+        "endprimitive ",
+        L(0, {"primitive", "mixed",
+              "(", "o", ",", "clk", ",", "j", ",", "k", ",",
+              "preset", ",", "clear", ")", ";"}),
+        NL(1, {"output", "o", ";"}),
+        NL(1, {"reg", "o", ";"}),
+        NL(1, {"input", "c", ";"}),
+        NL(1, {"input", "j", ",", "k", ";"}),
+        NL(1, {"input", "preset", ",", "clear", ";"}),
+        UdpBody(1, L(1, {"table"}),
+                   NL(2, {"?", "?", "?", "0", "1", ":", "?", ":", "1", ";"}),
+                   NL(2, {"?", "?", "?", "*", "1", ":", "1", ":", "1", ";"}),
+                   NL(2, {"?", "?", "?", "1", "0", ":", "?", ":", "0", ";"}),
+                   NL(2, {"?", "?", "?", "1", "*", ":", "0", ":", "0", ";"}),
+                   NL(2, {"r", "0", "0", "0", "0", ":", "0", ":", "1", ";"}),
+                   NL(2, {"r", "0", "0", "1", "1", ":", "?", ":", "-", ";"}),
+                   NL(2, {"r", "0", "1", "1", "1", ":", "?", ":", "0", ";"}),
+                   NL(2, {"r", "1", "0", "1", "1", ":", "?", ":", "1", ";"}),
+                   NL(2, {"r", "1", "1", "1", "1", ":", "0", ":", "1", ";"}),
+                   NL(2, {"r", "1", "1", "1", "1", ":", "1", ":", "0", ";"}),
+                   NL(2, {"f", "?", "?", "?", "?", ":", "?", ":", "-", ";"}),
+                   NL(2, {"b", "*", "?", "?", "?", ":", "?", ":", "-", ";"}),
+                   NL(2, {"b", "?", "*", "?", "?", ":", "?", ":", "-", ";"}),
+                   L(1, {"endtable"})
+               ),
+        L(0, {"endprimitive"}),
+    },
+};
```

Comment:
Suggest test cases:
* Some comments in the port section where `input` and `output` are declared.
* Some comments in the UDP body

I was going to suggest ANSI-style port declaration (`udp_declaration_port_list`), e.g. `primitive p(output o, input i, j, k);`, but it doesn't look like my grammar supports that yet.  I'll file a separate bug for that.

Reply:
Added some tests with comments.